src/nupic/types/Serializable.hpp

```diff
@@ -217,8 +217,8 @@ class Serializable {
 	//       Remove the following two lines.
 
   // These must be implemented by the subclass.
-  virtual void save(std::ostream &stream) const {};
-  virtual void load(std::istream &stream) {};
+  virtual void save(std::ostream &stream) const { saveToStream_ar(stream); };
+  virtual void load(std::istream &stream) { loadFromStream_ar(stream); };
```

Comment:
what is this global change? It probably should not be part of this PR

Reply:
No.  The objective is to keep save/load as-is until Cereal is complete and then remove them.  That is so that we don't have all serialization broken until we finish.  So, don't redirect to saveToStream_ar( ) and loadFromStream_ar( )

Reply:
I made this change so that my tests can call save/load.  Since I rewrote the serialization for these classes, they do not implement old save/load, only cereal.

Save/load are the public facing API, and eventually the plan is to  rename "saveToStream_ar" to "save".  This change renames them except for the classes which still implement the old methods.

Reply:
The old implementation of save/load must remain in place until Cerial is complete.  They don't mix correctly with save_ar and load_ar.

Reply:
If you have a class that does not have the original save/load then just don't implement them.